doc/usermanual/darkroom/modules/tone/tonecurve.xml

```diff
@@ -68,11 +67,74 @@
       below) at the selected spot or area are shown on top left of the widget.
     </para>
 
+    <sect5 status="draft">
+      <title>color spaces</title>
+      <para>
+        Depending on the desired intent, you car apply the tone curve in 3 different color spaces:
+        <ul>
```

Comment:
Please use <itemizedlist></itemizedlist> and <listitem></listitem>.

Reply:
done